UPGRADE-4.1.md

```diff
@@ -16,3 +16,10 @@ Translation
 
  * The `FileDumper::setBackup()` method is deprecated and will be removed in 5.0.
  * The `TranslationWriter::disableBackup()` method is deprecated and will be removed in 5.0.
+
+Workflow
+--------
+
+ * Deprecated the `add` method in favor of the `addWorkflow` method in `Workflow\Registry`
```

Comment:
As I asked a mandatory change, Can I also ask you to add a final dot on each line of each .md updated?